File: CS/dxSample/Custom Layout Control/CustomLayoutControlItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraLayout;

namespace dxSample {
    internal class CustomLayoutControlItem : LayoutControlItem {
        private bool _HighlightFocusedItemEx;
        public bool HighlightFocusedItemEx {
            get {
                return _HighlightFocusedItemEx;
            }
            set {
                if (HighlightFocusedItemEx == value) {
                    return;
                }
                _HighlightFocusedItemEx = value;
            }
        }
        public CustomLayoutControlItem(LayoutControlGroup parent)
            : base(parent) {

            _HighlightFocusedItemEx = parent.Owner.OptionsView.HighlightFocusedItem;
        }
    }
}

File: VB/dxSample/Custom Layout Control/CustomLayoutControlItem.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraLayout

Namespace dxSample
    Friend Class CustomLayoutControlItem
        Inherits LayoutControlItem

        Private _HighlightFocusedItemEx As Boolean
        Public Property HighlightFocusedItemEx() As Boolean
            Get
                Return _HighlightFocusedItemEx
            End Get
            Set(ByVal value As Boolean)
                If HighlightFocusedItemEx = value Then
                    Return
                End If
                _HighlightFocusedItemEx = value
            End Set
        End Property
        Public Sub New(ByVal parent As LayoutControlGroup)
            MyBase.New(parent)

            _HighlightFocusedItemEx = parent.Owner.OptionsView.HighlightFocusedItem
        End Sub
    End Class
End Namespace
